tests/testthat/_snaps/pluralization.md

```diff
@@ -176,3 +176,96 @@
     Output
       9 word
 
+# issue 701
+
+    Code
+      print(pluralize("{NA} file{?s} expected"))
+    Output
+      NA file expected
```

Comment:
Should we literally do `NA`, or something else?

```suggestion
      ? file expected
```

Reply:
This might need to wait for @gaborcsardi, because I wonder if missing values should be an error?

Reply:
I think we can process missing values, I would consider that a feature. Calling them NA is fine as well, people using R should be used to that.